ethereumClassic-cpp-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,203 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "storageAccountNamePrefix": {
```

Comment:
(it looks like we are taking this prefix, then manipulating it with replaces and uniqueString in the variables section; this is better than just having the parameter, but if we're already manipulating the string, why not just aut-generate it with uniqueString in the first place?)